package/gluon-geolocator/luasrc/lib/gluon/upgrade/540-geolocator-settings

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site'
```

Comment:
You switched to the new *gluon.site* library, but didn't adjust the rest of the code accordingly... see other comments below.